inc/config/query-context/query-context.php

```diff
@@ -160,4 +161,26 @@ public function get_query_name(): string {
 	public function get_query_runner(): QueryRunnerInterface {
 		return new QueryRunner( $this );
 	}
+
+	/**
+	 * Override this method to determine whether a response is a collection at runtime.
+	 *
+	 * @return bool
+	 */
+	public function is_collection(): bool {
```

Comment:
What is the use case for this `is_collection()` method override? Maybe the response needs to be inspected? In that case, it seems like it should get passed the `$response_data`? Or removed